modules/statics/src/coins.ts

```diff
@@ -1355,6 +1356,8 @@ export const coins = CoinMap.fromCoins([
     UnderlyingAsset.SLND,
     AccountCoin.DEFAULT_FEATURES
   ),
+  fiat('fiat', 'Fiat', UnderlyingAsset.USD, 2, Networks.main.fiat),
```

Comment:
Should the `UnderlyingAsset` here be USD? I imagine there should be a generic `UnderlyingAsset.FIAT` for this case, but maybe not? Not sure where that is used, so maybe its irrelevant for FIAT